pop-up.css

```diff
@@ -0,0 +1,209 @@
+* {
+  margin: 0;
+  padding: 0;
+  outline: 0;
+  border: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+html {
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+}
+
+body {
+  background: #eee;
+}
+
+#modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  min-height: 100vh;
+  width: 100%;
+  background: hsla(216, 14%, 79%, 0.695);
+  backdrop-filter: blur(6px);
+  display: none;
+  align-items: center;
+  justify-content: center;
+  z-index: 7;
+}
+
+.modal-container {
+  width: 90%;
+  margin: 0 auto;
+}
+
+#modal .popup {
+  padding: 0.5rem;
+  text-align: center;
+  background: #fff;
+  position: relative;
+  width: 80%;
+  overflow-y: auto;
+  height: 90vh;
+  margin: 0 auto 1rem;
+  border-radius: 0.5rem;
+  z-index: 5;
+}
+
+.project-popup .popup.active {
+  display: block;
+}
+
+.header-close {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 0.2rem;
+}
+
+.header-close h3 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 1.2rem;
+  color: #174d31;
+  padding-top: 0.4rem;
+}
+
+.card-info {
+  width: 100%;
+}
+
+.main-img {
+  height: auto;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.title-and-date {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 0.4rem;
+  margin-bottom: 0.8rem;
+  flex-wrap: wrap;
+}
+
+.title-and-date .title {
+  font-family: 'Poppins', serif;
+  font-weight: 600;
+  font-size: 0.8rem;
+  color: #344563;
+  font-style: normal;
+}
+
+.title-and-date span {
+  width: 0.4rem;
+  height: 0.4rem;
+  border-radius: 50%;
+  background-color: #c1c7d0;
+}
+
+.title-and-date .muted {
+  font-family: 'Poppins', serif;
+  color: #7a869a;
+  font-weight: 500;
+  font-size: 0.8rem;
+}
+
+.card-info .switch .description {
+  font-family: 'Poppins', serif;
+  font-weight: 400;
+  font-size: 0.8rem;
+  line-height: 23px;
+  text-align: left;
+  color: #344563;
+  margin-bottom: 0.8rem;
+}
+
+.card-info .description .hide {
+  display: none;
+}
+
+.card-info .stack-and-buttons .stack {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 0.3rem;
+  margin-bottom: 2.2rem;
+  flex-wrap: wrap;
+}
+
+.card-info .stack-and-buttons .stack li {
+  font-family: 'Poppins', serif;
+  padding: 0.25rem 0.75rem;
+  background: #ebebff;
+  border-radius: 0.3rem;
+  color: #6070ff;
+  font-weight: 400;
+  font-size: 0.75rem;
+}
+
+.card-info .switch .stack-and-buttons .stack .hide {
+  display: none;
+}
+
+.buttons {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 0.2rem 0;
+  flex-wrap: wrap;
+}
```

Comment:
- **[OPTIONAL]** Well done implementing the required changes by the previous CR, please 🙏🏽 correct the position of the buttons to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/233138964-e60f1cb8-960d-4f8d-8af7-fd0f5e40d40f.png) | ![image](https://user-images.githubusercontent.com/58642949/233138858-a9ccd6df-4afe-4645-b556-20459aff455a.png) |